Plenigo 1175 add mobile token functionality

SebaSOFT opened the pull request:
- First (unchecked, uncommented) version of the shortcode
- Changelog and version 1.61
- Breaking in the first AppID found for a customer/product
- Fixed some bugs, now showing the list and actually triggering the creation/deletion of the App ID

Test site:   <url>/?page_id=279

Commit:
- First (unchecked, uncommented) version of the shortcode

Commit:
- Changelog and version 1.61

Commit:
- Breaking in the first AppID found for a customer/product

Commit:
- Fixed some bugs, now showing the list and actually triggering the creation/deletion of the App ID

Commit:
- Token instead of APP ID
- Some corrections upon parameter receive
- Still missing the text field.

Commit:
- Mobile Token now obtained with Device name and all
- SImple Javascript validation

Commit:
- Updated SDK

maximilianschweitzer commented on plenigo_plugin/PlenigoShortcodeManager.php:
```diff
@@ -309,6 +320,191 @@ public function plenigo_handle_user_shortcode($atts, $content = null, $tag = nul
+        $res = '';
+        $customerID = $user->getId();
+        $arrProducts = UserService::getProductsBought($customerID);
+        //FIXME TODO for subscriptions
```
Would be cool if it would also work for subscriptions. ;-)

maximilianschweitzer commented on plenigo_plugin/PlenigoShortcodeManager.php:
```diff
@@ -309,6 +320,191 @@ public function plenigo_handle_user_shortcode($atts, $content = null, $tag = nul
+                $hasModified = false;
+            }
+
+            /* if ($hasModified) { // Reload Array
```
commented code

SebaSOFT replied:
Done

Commit:
- Subscriptions added to the array (untested)

Commit:
- Removing commented code

Commit:
- Compatible with WP 4.5.3

Commit:
- Solved having to refresh the screen after deleting an app id
- Solved blank confirmation dialog upon deletion

Commit:
- endDate check for subscriptions

Commit:
- Multiple AppIDs per product
- Removing a particular AppID instead of all AppIDs from a product
- Create as many tokens as can be redeemed.
- Date check for subscriptions
- hasBought check for Product

Commit:
- Fix for a bug on IE 10 and Edge browsers
- Change <br> for <div>'s with the "plenigoMToken" and "plenigoTokenCreate" classes, so it can be stylized for any site

Commit:
- Translations for the Mobile Token Administration interface
